python/graphstorm/model/embed.py

```diff
@@ -290,6 +344,8 @@ def forward(self, input_feats, input_nodes):
         assert isinstance(input_nodes, dict), 'The input node IDs should be in a dict.'
         embs = {}
         for ntype in input_nodes:
+            if isinstance(input_nodes[ntype], np.ndarray):
```

Comment:
Why input_nodes is an numpy ndarray?

Reply:
It could be a ndarray, regarding an existing test: https://github.com/awslabs/graphstorm/blob/5e7cbb8b1e781fb90959f280c47bd84b4c429508/tests/unit-tests/test_embed.py#L67
It's fine if it's a DGL distTensor/PyT tensor (indexing array is a ndarray) but it wouldn't work for wholegraph gather, which requires to be a tensor obj... I can move this line under the condition `if is_wholegraph_embedding_module(..):` if you prefer.